atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1283,7 +1284,12 @@ public void putMetadataForTables(final Map<TableReference, byte[]> tableRefToMet
         internalPutMetadataForTables(tableRefToMetadata, true);
     }
 
-    private void internalPutMetadataForTables(final Map<TableReference, byte[]> tableNameToMetadata, final boolean possiblyNeedToPerformSettingsChanges) {
+    private void internalPutMetadataForTables(final Map<TableReference, byte[]> unfilteredTableNameToMetadata, final boolean possiblyNeedToPerformSettingsChanges) {
```

Comment:
Can we add a test for this / someway ensure that this isn't going to cause problems for tables that want empty metadata entries that will now simply not exist.  Not sure how common a case this is and I understand the urgency, just seems like it wouldn't fit the general KVS contract.